nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the install medium {#sec-installation-booting}
+
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Plug in the install drive. Then turn on or restart your computer.
+
+2.   Press <kbd>F12</kbd> (or the appropriate key) to open the boot menu.
+     Select the USB flash drive (the option usually contains the word "USB").
+     If you choose the incorrect drive, your computer will likely continue to
+     boot as normal. In that case restart your compute and pick a different drive.
```

Comment:
```suggestion
     If you choose the incorrect drive, your computer will either continue to
     boot as normal or abort with no OS found. In both cases restart your compute and pick a different drive.
```

Reply:
In addition to suggestions, *your comput* → *your compute**r***.

Reply:
i think that is a rare case and does not add much useful information. the "likely" says that other things can happen, but they are not relevant to mention here